src/App.tsx

```diff
@@ -15,10 +67,11 @@ export const App: React.FC = () => {
 
       <div className="todoapp__content">
         <header className="todoapp__header">
-          {/* this buttons is active only if there are some active todos */}
-          <button type="button" className="todoapp__toggle-all active" />
+          {todos.length && (
```

Comment:
Replace 'todos.length' with 'todos.length > 0' to prevent rendering '0' when no todos are present.